app/models/file_search.rb

```diff
@@ -72,46 +94,59 @@ def year_range
     all_years
   end
 
-  def results
-    SearchResults.new(files)
-  end
-
   def files
-    params.empty? ? highlighted_files : filtered_files
+    query = {}
+    query[:q] = search_term if search_term
+    query[:f] = filters unless filters.empty?
+    query[:page] = page if page
+
+    (response, documents) = catalog_query.search_results(query, catalog_query.search_params_logic)
+    if @response = response
+      @total_items = @response.response["numFound"]
+      @total_pages = @response.total_pages
+      @current_page = @response.current_page
+      @next_page = @response.next_page
```

Comment:
Do these need to be instvars?  Since `@response` already is, could we create getter methods for these instead?